lib/queuetopia.ex

```diff
@@ -112,12 +112,23 @@ defmodule Queuetopia do
         result =
           Queuetopia.Jobs.create_job(@repo, @performer, @scope, queue, action, params, opts)
 
-        with {:ok, %Job{}} <- result,
-             scheduler_pid when is_pid(scheduler_pid) <- Process.whereis(scheduler()) do
-          Queuetopia.Scheduler.send_poll(scheduler_pid)
+        case result do
+          {:ok, %Job{}} ->
+            send_poll()
+
+            result
+
+          _ ->
+            result
         end
+      end
 
-        result
+      def send_poll() do
+        with scheduler_pid when is_pid(scheduler_pid) <- Process.whereis(scheduler()) do
+          Queuetopia.Scheduler.send_poll(scheduler_pid)
+
+          :ok
```

Comment:
I would like this function to return :ok out of the with statement, so the return value will always be :ok.

Reply:
If you want to handle the scheduler down status, you alternatively do it explicitly, returning {:error, "scheduler down"}